appcues/src/main/java/com/appcues/ui/composables/AppcuesComposition.kt

```diff
@@ -158,8 +164,16 @@ private fun BoxScope.ComposeLastRenderingState(state: Rendering) {
     }
 }
 
+@OptIn(ExperimentalComposeUiApi::class)
 @Composable
 private fun BoxScope.ApplyBackgroundDecoratingTraits(list: List<BackdropDecoratingTrait>) {
+    // adds a layer behind all Backdrop decorating traits to prevent touch events to
+    Spacer(
+        modifier = Modifier
+            .fillMaxSize()
+            .pointerInteropFilter { false }
+    )
```

Comment:
wondering if this needs to be pulled out to the caller? this method recursively calls itself on line 180 below, so this would actually add N Spacers each time through the recursion between background decorations, if multiple exist?

Reply:
totally